docs/tools-and-applications/airflow/airflow.md

```diff
@@ -0,0 +1,28 @@
+
+# Airflow
+
+## What is it?
+
+[**Airflow**](https://airflow.apache.org/) is a platform created by the community to programmatically author, schedule and monitor workflows.  
```

Comment:
Can you add a nice Logo, like for Spark?

Reply:
Added